languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -15787,6 +15787,127 @@ USA
+                <token min="0" max="2" postag_regexp="yes" postag="MD|RB" />
+                <token min="0" inflected="yes">do</token>
+                <token>see</token>
+                <token min="0" postag="PCT" />
+                <token>it</token><!-- subject or expletive -->
+            </antipattern>
+            <antipattern>
+                <token postag_regexp="yes" postag="SENT_START|PCT|CC|WP" />
+                <token postag="VBG" />
+            </antipattern>
+            <antipattern>
+                <token postag_regexp="yes" postag="SENT_START|PCT|CC|WP" />
+                <token inflected="yes">have</token>
+            </antipattern>
+            <antipattern>
+                <token postag="VB.*" postag_regexp="yes" inflected="yes" regexp="yes">get|see</token>
+                <token />
+                <token postag="NNS" />
+            </antipattern>
+            <antipattern>
+                <token>her</token><!-- 'her' is ambiguously PRP$ -->
+                <token postag="NNS" />
+            </antipattern>
+            <antipattern>
+                <token inflected="yes">make</token>
+                <token regexp="yes">us|them</token>
+                <token postag="NNS" />
+            </antipattern>
+            <antipattern>
+                <token>for</token><!-- when I ask for help it starts working -->
+                <token>help</token>
+            </antipattern>
+            <antipattern>
+                <token regexp="yes">thanks|guys</token>
+            </antipattern>
+            <antipattern>
+                <token postag="PRP" />
+                <token>does</token>
+                <token postag="VB" />
+            </antipattern>
+            <antipattern>
+                <token postag="WP"><exception regexp="yes">which|that</exception></token>
+                <token min="0" max="3" postag_regexp="yes" postag="RB|VB|TO" />
+                <token />
+            </antipattern>
+
+            <pattern>
+                <token postag="VB.*" postag_regexp="yes" inflected="yes" regexp="yes">make|let|get|help|see|have
+                    <!--<exception scope="previous" postag="DT" />-->
+                </token>
+                <token postag="PRP" regexp="yes">me|you|him|her|it|them|us</token>
+                <!--<token min="0">does</token>--> <!-- not sure how to greedily match -->
+                <marker>
+                    <token postag="VBZ" case_sensitive="yes" regexp="yes">[a-z]+
+                    <!--<token postag="VBZ" regexp="yes">-->
+                        <exception postag="VB" />
+                        <!--<exception inflected="yes">be</exception>-->
+                        <exception regexp="yes">is|isn</exception>
+                    </token>
+                </marker>
+            </pattern>
+            <message>The bare infinitive <suggestion><match no="3" postag="VB" /></suggestion> should follow the verb '\1' and the object '\2'.</message>
```

Comment:
Not to be nit-picky, but doesn't this inverted syntax focus "follow"? Maybe a passive construction would be clearer...
"The verb ... and the object ... should be followed by the bare infinitive ... ."